pybalance/utils/tests/test_balance_calculators.py

```diff
@@ -241,3 +241,49 @@ def test_gamma_is_area_between_cdfs():
         )
 
     assert np.isclose(distance_popmat, distance_direct, atol=1e-3)
+
+
+def test_target_subsets():
+
+    matching_data = generate_toy_dataset()
+    target, pool = split_target_pool(matching_data)
+    gamma = BalanceCalculator(matching_data, "gamma", n_bins=20)
+
+    d1 = gamma.distance(pool)
```

Comment:
ummm so you've removed the global scope self.target... so i'm unclear what distance does if there is no target? i.e. what happens when target_subsets = None?